dev-tools/docker_build.sh

```diff
@@ -67,6 +67,9 @@ cd "$TOOLS_DIR/.."
 # necessary network access
 3rd_party/pull-eigen.sh
 
+du -sh .
+df -h .
```

Comment:
Was this just temporary debug that can be removed now?

Or if you think this would be useful to keep forever then please add some `echo`s with titles for the output.

Reply:
> Was this just temporary debug that can be removed now?

Yes, just temporary while I was trying to correctly size the VM's disk. I'll remove it.